examples/src/main/kotlin/usageExamples/update/Update.kt

```diff
@@ -0,0 +1,55 @@
+package usageExamples.update
+
+// :replace-start: {
+//    "terms": {
+//       "CONNECTION_URI_PLACEHOLDER": "\"<connection string uri>\"",
+//       "import io.github.cdimascio.dotenv.dotenv\n": ""
+//    }
+// }
+// :snippet-start: update-usage-example
+
+import com.mongodb.MongoException
+import com.mongodb.client.model.Filters
+import com.mongodb.client.model.UpdateOptions
+import com.mongodb.client.model.Updates
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.coroutines.runBlocking
+import java.util.*
+
+data class Movie(val title: String, val runtime: Int, val genres: List<String>, val lastUpdated: Date)
```

Comment:
nit: can you make this data class go across multiple lines so the later values aren't cut off

Reply:
Updated